RATapi/project.py

```diff
@@ -130,13 +131,25 @@ class Project(BaseModel, validate_assignment=True, extra="forbid"):
     inputs required for a reflectivity calculation.
     """
 
+    model_config = ConfigDict(use_attribute_docstrings=True)
+
     name: str = ""
+    """The name of the project."""
+
     calculation: Calculations = Calculations.Normal
+    """What calculation type should be used. Can be 'normal' or 'domains'."""
+
     model: LayerModels = LayerModels.StandardLayers
+    """What layer model should be used. Can be 'standard layers', 'custom layers', or 'custom xy'."""
+
     geometry: Geometries = Geometries.AirSubstrate
+    """What geometry should be used. Can be 'air/substrate' or 'substrate/liquid'"""
+
     absorption: bool = False
+    """Whether imaginary (absorption) SLD should be accounted for."""
 
     parameters: ClassList[RATapi.models.Parameter] = ClassList()
+    """The list of parameters used in the layers of a model."""
```

Comment:
I think the following strings should also start with "The list of parameters . . ." rather than "The parameters . . ."